build_scripts/release_scripts/deploy.sh

```diff
@@ -31,20 +31,18 @@ if [[ ! -f "$JAR_FILE" ]]; then
 fi
 
 mvn deploy:deploy-file -Dfile=$JAR_FILE -DgroupId=com.mysql.ndb -DartifactId=clusterj-rondb \
```

Comment:
this file is only for x86_64. add a check so that it is not called for any other architecture

Reply:
See my other comment regarding the Jar files (i.e. why Java only for x86?)

Reply:
Have a look now!